### ISSUE TITLE
Handler example

### ISSUE THREAD

**Issue by manuel-palacio** [mistic100/angular-jqcloud#1]
Could you add an example on how to use a click handle? This seems a bit awkward when I tried. I could not call another method in my controller from it.

**Comment by mr-White** [comment]
Not true. You can access angular stuff from the outside even though it is not a "best practice"

Here is an example:
```javascript
handlers: {
   click: function() {
      angular.element('#body-id').scope().someFunctionOnThatScope()
   }
}
```

**Comment by mjl1** [comment]
By combining a few random ideas and suggestions, here's a snippet that is able to call a function in my Controller. ($scope.themes is my array of words, gathered from a ThemeMap object). For the click handler I create a copy of the val and use $scope.$apply on my controller's function:


                $scope.themes = [];
                for (var val in $scope.ThemeMap) {
                    $scope.themes.push({
                        text: val,
                        weight :$scope.ThemeMap[val].length,
                        handlers:
                            {click: function() {
                                var newVal = angular.copy(val)
                                return function() {
                                   $scope.$apply($scope.checkByTheme(newVal))
                               }
                            }()
                            }
                        })
                }
